libstb/secvar/backend/edk2-compat-process.c

```diff
@@ -25,8 +25,13 @@ int update_variable_in_bank(struct secvar *secvar, const char *data,
 	struct secvar_node *node;
 
 	node = find_secvar(secvar->key, secvar->key_len, &staging_bank);
-	if (!node)
-		return OPAL_EMPTY;
+	if (!node) {
+		node = new_secvar(secvar->key, secvar->key_len,data,  dsize, 0);
+		if (!node)
+			return OPAL_NO_MEM;
+		list_add_tail(&staging_bank, &node->link);
+		goto out;
+	}
```

Comment:
Are we just implicitly creating the empty variables in the staging bank here? This seems a bit strange when we are allocating them in `.pre_process()` as well, only to completely ignore them here.